test/Microsoft.DocAsCode.MarkdigEngine.Tests/CodeTest.cs

```diff
@@ -0,0 +1,603 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Tests in `MarkdigEngine.Extensions.Tests` project are shared between docfx v2 and docfx v3, while tests in `MarkdigEngine.Tests` only covers docfx v2. Consider move this test to `MarkdigEngine.Extensions.Tests` unless it is tightly coupled with v2.

Reply:
@yufeih - fantastic catch! I moved and corrected the tests!